src/worldofzuul/CommandWords.java

```diff
@@ -7,31 +7,51 @@
  * @version 2006.03.30
  */
 
+
 public class CommandWords {
     private HashMap<String, CommandWord> validCommands;
```

Comment:
Attributen skal også have en docblock der beskriver hvad variablen indeholder.

Reply:
Fixed in b755d8f12058684debf06e1063375fde3b49ae57